config/default.yml

```diff
@@ -109,6 +109,19 @@ PreCommit:
     flags: ['-EIHn', '(it|describe)\.only']
     include: 'test/**/*.js'
 
+  Spectral:
+    enabled: false
+    description: 'Analyze with Spectral'
+    requiredExecutable: './node_modules/.bin/spectral'
+    globalRequiredExecutable: 'spectral'
+    flags: ['lint', '--quiet', '--ignore-unknown-format', '--format', 'text']
+    installCommand: 'npm install @stoplight/spectral'
+    globalInstallCommand: 'npm install -g @stoplight/spectral'
+    include:
+      - '**/*.js'
+      - '**/*.yml'
```

Comment:
It would be nice to lint Affiance's yml files with this pre commit hook to help these default config files stay consistent.